{
  "id": 3877636,
  "key": "MM-T403",
  "name": "Email address already in use",
  "createdOn": "2020-01-03T00:15:37Z",
  "objective": null,
  "precondition": null,
  "estimatedTime": null,
  "labels": [],
  "component": null,
  "priority": { "id": 401951, "name": "Low" },
  "status": { "id": 890281, "name": "Active" },
  "folder": {
    "id": 1341388,
    "name": "Automated tests",
    "parentId": 883991,
    "fullNames": ["Cloud", "Automated tests"],
    "fullPath": "cloud/automated-tests"
  },
  "customFields": {
    "Fix Versions": [],
    "Playwright": null,
    "Authors": "",
    "Priority P1 to P4": "P2 - Core Functions (Do core functions work?)",
    "Update notes": "Steve Mudie — Monday, December 21, 2020 3:19 PM<br>• Added link to existing Cypress code in Traceability tab above<br>• Code was put in place prior to shift over to using the test case numbers ➜ \"it('On14634 Email address already exists', () =&gt; {\"<br>• Added to manual pool until automation can be rectified<br>• Set priority to P2<br>-----------------------------------------------<br>Prapti Shrestha - Thursday, April 16, 2021<ul><li>Updated Zephyr key in E2E test file.</li><li>Took off `TM4J-Key-Missing-In-Cypress` label</li><li>Set Cypress status to Production</li></ul><br>",
    "Rainforest": ["N/A"],
    "Cypress Reports": "MISSING - TM4J-Key-Missing-In-Cypress - TM4J key NOT found in any Cypress run reports  (SDET/QA)",
    "Team Ownership": ["Growth"],
    "Manual Test Environments": ["Automated - No manual tests needed"],
    "Detox": null,
    "Cypress": "in Production",
    "MMCTL": null,
    "Last Updated": "",
    "Selenium": null,
    "Tags": [],
    "Location": "Cloud"
  },
  "steps": [
    {
      "inline": {
        "description": "Email address already in use<br>–––––––––––––––––––––––––<ol><li>After creating a new user, log out of that server</li><li> Click to Create a new user</li><li> Enter the same email address you already used to create an account</li><li>Enter any username and any password and click to create the account</li></ol>",
        "testData": null,
        "expectedResult": "<ul><li>Error message displays below the Create Account button that says \"<em>An account with that email already exists</em>\"</li><li>From there you can just click the Back link in top left and abandon that account creation</li></ul>",
        "customFields": {
          "Fix Versions": [],
          "Related ticket(s)": "",
          "Important notes about this step": ""
        }
      },
      "testCase": null
    }
  ],
  "caseHashed": "9c3ebedd2a053fc34a4123d363a995507b945bb398c421147640853b2d2b55f53595a7b59c849a5621be240157c25443",
  "stepsHashed": "2f9d01b83ff30b1cb3473e62ef86920162840e35bbf5a42f3ce95777cc038792d28abfe1d5d849e32d47aa37d88e4c59"
}
